build.md

```diff
@@ -0,0 +1,41 @@
+# Build luet packages for kairos
+
+kairos uses github action to build packages and push it to quay.io/kairos/packages(amd64 version) or quay.io/kairos/packages-arm64 (arm64 version),
+in case you wanna build it manually, e.g. testing a new added package locally, or create your own repo to serve it offline/privately, it can be done with this steps:
+
+## build packages
+
+To build all packages, go repo root dir, and issue commands like this (note, you need to run as root user, otherwise you will get some permission issues):
+
+```shell
+sudo luet build --all
+```
+
+To build an individual package, e.g. k8s/k3s-openrc
+
+```shell
+sudo luet build k8s/k3s-openrc
+```
+
+## Create repo
+
+After packages are build, you may create your repo using luet command, e.g. to create a docker repo for hosting your packages,
+
+```shell
+luet create-repo --type=docker  --output=myorg.org/packages
```

Comment:
you can pass `--push-images` here and it will autopush the images and generate and push the repo files, so you can skip the next step